torch/onnx/_internal/fx/symbolic_exporter.py

```diff
@@ -209,7 +211,7 @@ def export_without_parameters_and_buffers(
     return (
         exporter._export(
             graph_module,
-            (*bound_args, *replaced_attrs),
+            *(*bound_args, *replaced_attrs),
```

Comment:
as commented above; but if we really want this:

```suggestion
            *bound_args,
            *replaced_attrs,
```